data-visualization/data-visualization.md

```diff
@@ -2,5 +2,30 @@
 layout: page
 title:  Data visualization
 category: Data visualization
+published: true
 ---
 
+Whether you are using Excel, Tableau, D3.js, or any other program, follow
```

Comment:
Yep! I'll break your suggestion up a bit to read `These guidelines make the most of effective and consistent data visualizations produced by the CFPB. They are meant to be platform agnostic, and aim to incorporate current usage of Excel, Tableau, and D3.js as well as the future use of any other program.`